data/korath/korath weapons.txt

```diff
@@ -1011,6 +1011,36 @@ effect "firelight ring"
 	"velocity scale" 0.1
 
 
+outfit "Microbot Factory"
+	category "Guns"
+	thumbnail "outfit/unknown"
+	"mass" 182
+	"outfit space" -212
+	"weapon capacity" -112
+	"required crew" 20
+	"gun ports" -1
+	"spinal mount" -1
```

Comment:
I think this should lock from both directions. That is, while this has -1 and the ship has 1 spinal mount, there should be some other attribute where this has 1 and the ship has -1.

Reply:
It's intentional for it to be removable

Reply:
I agree with Ravenshining: I don't see a need for this to be locked in.

Reply:
There are no opportunities to cap this ship and take it to an outfitter at present, anyway.  Between that and it being `unplunderable` it's already locked to the ship.

In the future, however, if we ever get an implementation of #2004 , plus a facility to make hardpoint sprites appear *under* a ship, and some artistic revision, we could remove that tag and see this mounted on the other spinal ships as we originally intended.